Add Cargo-features for `tokio` version `0.2` (#1180)
serenity-rs/serenity: README.md

The default features are: `builder`, `cache`, `client`, `framework`, `gateway`,
`http`, `model`, `standard_framework`, `utils`, and `rustls_backend`.

There are these alternative default features, they require to set `default-features = false`:

- **default_tokio_0_2**: Uses the default backend with `tokio` version `0.2`.
- **default_native_tls**: Uses `native_tls_backend` instead of the default `rustls_backend`.
- **default_native_tls_tokio_0_2**: Uses `native_tls_backend` with `tokio` version `0.2`.
- **default_no_backend**: Excludes the default backend, pick your own backend instead.

> If you are unsure which to pick, just go with the normal default features (by not setting `default-features = false`).

Reviewer: ```suggestion
If you are unsure which to pick, use the default features by not setting `default-features = false`.
```

- **native_tls_backend**: Uses SChannel on Windows, Secure Transport on macOS,
and OpenSSL on other platforms.

> In case you need the old `tokio` version `0.2`, pick one of these backends instead:

Reviewer: ```suggestion
If you need to use `tokio` version `0.2` use the backends below:
```
